src/MinimalDelegation.sol

```diff
@@ -32,23 +35,33 @@ contract MinimalDelegation is IERC7821 {
 
     /// @dev Authorizes the `key`.
     function authorize(Key memory key) external returns (bytes32 keyHash) {
-        keyHash = key.hash();
-        MinimalDelegationStorageLib.get().keyStorage[keyHash] = abi.encode(key);
+        keyHash = _authorize(key);
```

Comment:
this is scary, I guess we merged in the other authorize/revoke PR without checking the msg.sender 😨 . Can you add auth checks to `authorize` and `revoke`?